.gitignore

```diff
@@ -44,5 +44,7 @@ docker
 fly.toml
 
 # Ignore extensions (post installable extension PR)
-extensions/
-upgrades/
+lnbits/extensions/*
+!lnbits/extensions/pyproject.toml
+!lnbits/extensions/extensions/
```

Comment:
i do not like the fact that i uses extensions/extensions dir, cant we just but the first pyproject inside lnbits/ and use extensions dir as is?

Reply:
This would work. I had it this way because poetry requires the directory to exist, but I think you're right. It could even be in the root folder aswell